client/jetpack-app/page-middleware/layout.jsx

```diff
@@ -0,0 +1,53 @@
+import { QueryClientProvider } from '@tanstack/react-query';
+import { Provider as ReduxProvider } from 'react-redux';
```

Comment:
Can you try reverting all **new files** to tsx? Did you try this before? Were there any changes that you encountered?

Reply:
Okay, I can try to convert them all to `tsx`. I haven't tried it before 👍